backend/django_core/static_src/components/minor/Comment/CommentBlock.svelte

```diff
@@ -231,7 +243,7 @@
         {/if}
 
         <!-- Render replies here -->
-        {#if item.childrens !== 0 && !item.collapse}
+        {#if !_.isEmpty(item.child) && item.childrens !== 0 && !item.collapse}
             <div class="mt-5 flex flex-col gap-5 md:mt-[1.5vw] md:gap-[1.5vw]">
                 {#each item.child as comment, index}
                     <svelte:self
```

Comment:
You see there's a `<svelte:self></svelte:self>` tag here?

You should have attached the dispatcher here too